docs/_ug/commandSummary/tagCommands/tag.md

```diff
@@ -1,2 +1,2 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>tag n/TAG_NAME id/INDEX_LIST</b><br><br><u>Valid Examples:</u><br>tag n/Condiments id/1<br>tag n/Condiments id/1,2,4,8<br><br><u>Invalid Example:</u><br>tag Condiments 1
+`tag n/TAG_NAME id/INDEX`
```

Comment:
I thought we are standardising to remove the `id/` prefix entirely?

```suggestion
`tag n/TAG_NAME INDEX`
```

Reply:
Oh, I realise our tag impelementation is still using id/. We'll put id/ for now until we change up the implementation and testing. Will update this after implementation is updated.